src/web/modules/transactions/update/upload.njk

```diff
@@ -106,4 +106,19 @@
     and 'reference' fields populated.
   </p>
 
-{% endblock %}
\ No newline at end of file
+  <h2 class="govuk-heading-s">Things to remember</h2>
+  <p class="govuk-body">
+    If a payment or refund is changed from error to success or vice-versa, the <code>refund_status</code>,
```

Comment:
This looks useful to give people context on this issue, we have some basic rules for what makes a "valid" CSV and it would be good to extend them to include this requirement formally. 

There is a ticket to track this, if this scenario could be added to it/ `+1`d `PP-6497`